Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\n# as it contains information specific to your local configuration.\n#\n# Location of the SDK. This is only used by Gradle.\n# For customization when using a Version Control System, please read the\n# header note.\n#Tue Jan 06 08:49:49 CET 2026\nsdk.dir=/Users/lukasbroberg/Library/Android/sdk\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision eeb933fe175f1169774f392bf9cedcc67aa68eda)
+++ b/local.properties	(date 1768430023393)
@@ -5,4 +5,4 @@
 # For customization when using a Version Control System, please read the
 # header note.
 #Tue Jan 06 08:49:49 CET 2026
-sdk.dir=/Users/lukasbroberg/Library/Android/sdk
+sdk.dir=/Users/simonmarsengo/Library/Android/sdk
